pkgs/development/tools/udis86/default.nix

```diff
@@ -0,0 +1,31 @@
+{ stdenv, fetchFromGitHub, autoreconfHook, python }:
+
+stdenv.mkDerivation rec {
+  pname = "udis86";
+  version = "1.7.2";
+
+  src = fetchFromGitHub {
+    owner = "vmt";
+    repo = "udis86";
+    rev = "v${version}";
+    url = https://github.com/vmt/udis86/archive/v1.7.2.tar.gz;
```

Comment:
Please replace `v1.7.2.tar.gz` with `v${version}.tar.gz` so we don't run into problems when someone forgets to update the version in 2 places.

Reply:
Done.